site/_posts/2017-04-24-Squashing-Bugz.md

```diff
@@ -0,0 +1,100 @@
+
+[Šimon](https://github.com/isimluk) Speed up certain seeding methods
+([#14721](https://github.com/ManageIQ/manageiq/pull/14721)) & ([#14712](https://github.com/ManageIQ/manageiq/pull/14712))
+
+
+## Squashing Bugz
+
+With the amount of new features introduced in last few sprints into ManageIQ, it is possible
+that we introduced new bugz as well as existing bugz were discovered.
+
+Several fixes were made in Embedded Ansible area and Ansible Services area, see list of PRs in manageiq-ui-classic & manageiq repos for full details on these.
+
+[MartinH](https://github.com/himdel) made performance improvements with Notifications ([#611](https://github.com/ManageIQ/manageiq-ui-classic/pull/611))
+
+[Brian](https://github.com/bmclaughlin) Fix Group/Role name links on User Details page ([#655](https://github.com/ManageIQ/manageiq-ui-classic/pull/655))
+
+[Aparna](https://github.com/AparnaKarve) Quadicon links fixed on Tagging edit screen ([#1041](https://github.com/ManageIQ/manageiq-ui-classic/pull/1041))
+
+[Zahi](https://github.com/zakiva) Add Container entities to TAG_CLASSES ([#14535](https://github.com/ManageIQ/manageiq/pull/14535/files))
+
+There were many more bugz that were fixed last week, please see complete list of merged PRs in each repo for more details.
+
+
+## Features
+
+While all our developers are heads down with squashing bugz some of us managed to  squeeze in some minor Enhancements this past week.
+
+[Jillian](https://github.com/jntullo) has added API support for Refresh Configuration Script Sources action
+([#14714](https://github.com/ManageIQ/manageiq/pull/14714)).
+
+[James](https://github.com/jameswnl) Added support to pickup Tower Project's refresh status 
+([#14687](https://github.com/ManageIQ/manageiq/pull/14687)).
+
+[Milan](http://github.com/mzazrivec) added UI support for Embedded Ansible refresh
+([#934](https://github.com/ManageIQ/manageiq-ui-classic/pull/934))
+
+[MartinP](http://github.com/martinpovolny) Custom buttons for list views
+([#796](https://github.com/ManageIQ/manageiq-ui-classic/pull/796))
+
+[Joshua](http://github.com/jjlangholtz) SUI - Show all custom actions vs collapsing to single dropdown
+([#669](https://github.com/ManageIQ/manageiq-ui-service/pull/669))
+
+[Allen](http://github.com/AllenBW) SUI - Power buttons vm details
+([#635](https://github.com/ManageIQ/manageiq-ui-service/pull/635))
+
+
+## Cleanup
+
+Aside from some new additions & lots of bug fixing, we also deleted some unwanted code from our repos.
+
+[MartinP](https://github.com/martinpovolny) Remove duplicate flash_msg_div
+([#1015](https://github.com/ManageIQ/manageiq-ui-classic/pull/1015)) & 
+([#1016](https://github.com/ManageIQ/manageiq-ui-classic/pull/1016)) 
+
+[Brandon](https://github.com/bdunne) Removal of unsused Global Variables from manageiq-ui-repo ([#969](https://github.com/ManageIQ/manageiq-ui-classic/pull/969))
```

Comment:
@imtayadeway this link is correct, it is supposed to be a link to a PR in manageiq-ui-classic repo: https://github.com/ManageIQ/manageiq-ui-classic/pull/969